### PR TITLE
global config: add three global options needed for logstore

### PR DESCRIPTION
**Pull request by norberttak**
This PR is about add three global options.
Signed-off-by: norberttakacs <norbert.takacs@balabit.com>

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

File: lib/cfg.c [hunk 1]
```diff
@@ -447,6 +447,10 @@ cfg_new(gint version)
   self->use_uniqid = FALSE;
   self->jvm_options = NULL;
 
+  self->logstore_journal_shmem_threshold = 536870912;
```
Comment (furiel) [inline lib/cfg.c]: We could write 512 * 1024 *1024. It would be a little more cleaner.

Comment (norberttak) [reply lib/cfg.c]: done

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by bazsi** [comment]
This should probably be reconsidered after #1885

**Comment by norberttak** [comment]
close this PR and continue the work on #1885

**Pull request closed without merging**